### ISSUE TITLE
Move the tiddlywiki.com documentation macros and styles into a plugin

### ISSUE THREAD

**Issue by Jermolene** [Jermolene/TiddlyWiki5#6852]
There is an extensive collection of useful macros and CSS styles that are used in the documentation on tiddlywiki.com. They are fairly regularly maintained and enhanced, and have remained backwards compatible.

The macros are documented here:

https://tiddlywiki.com/#Documentation%20Macros

It is proposed to move the macros and styles into a separate plugin called something like "tiddlydocs" so that it is easier for anyone to use it in their own wikis.

**Comment by kookma** [comment]
Absolutely! These are great tools and worth to be documented like https://tiddlywiki.com/#Documentation%20Macros

Also there was an .if macro: $:/editions/tw5.com/if-macro.js

**Comment by Jermolene** [comment]
> Also there was an .if macro: $:/editions/tw5.com/if-macro.js

Thanks @kookma, that really should be removed. I've made a ticket: https://github.com/Jermolene/TiddlyWiki5/issues/6854
